delivery_postlogistics/__manifest__.py

```diff
@@ -0,0 +1,41 @@
+# © 2013 Yannick Vaucher (Camptocamp SA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "PostLogistics Shipping - “Barcode” web service",
+    "summary": "Print PostLogistics shipping labels using the Barcode web service",
+    "version": "18.0.1.0.0",
+    "author": "Camptocamp,Odoo Community Association (OCA)",
+    "maintainer": "Camptocamp",
+    "license": "AGPL-3",
+    "category": "Delivery",
+    "complexity": "normal",
+    "depends": [
+        "stock_delivery",
+        "delivery_carrier_info",
+        "delivery_carrier_option",
+        "delivery_carrier_shipping_label",
+    ],
+    "website": "https://github.com/OCA/delivery-carrier",
+    "data": [
+        "security/ir.model.access.csv",
+        "data/partner.xml",
+        "data/product.xml",
+        "data/delivery.xml",
+        "data/package_type.xml",
+        "views/delivery.xml",
+        "views/stock_package_type_view.xml",
+        "views/stock_quant_package_view.xml",
+        "views/postlogistics_license.xml",
+        "views/res_partner_view.xml",
+        "views/stock.xml",
+    ],
+    "installable": True,
+    "auto_install": False,
+    "application": True,
+    "external_dependencies": {
+        "python": [
+            "openupgradelib",
+            "vcrpy",
```

Comment:
@yankinmax vcrpy IS NOT a dep. It's needed only for tests.
Also, please try to respect the commit msg guidelines.
I'm fixing these issues now and merge. I'm also dropping the change on the index.html file. 
TIP for devs: `export SKIP=oca-gen-addon-readme` to disable pre-commit readme update.

@leemannd commit msg issue applies to you as well :wink:

Reply:
Hello @simahawk, I've added vcrpy, because another PR build was failing without this dependency.
I'll check the commit message guideline. I thought I was respecting it.
Thanks for your help on this.